.travis.yml

```diff
@@ -4,11 +4,31 @@ env:
   global:
   - secure: TP9V3mSAnVJuzYmgAC2UOwXTCdp3aTGKf7i8hDqAZKe4WkVdSTLqksz4Z1/Ii5kEk3UrX46D0NjTy0wPLTRgHYdqybp1psQL394gw9VPADT/ZrZR1jl+MmqHQhg8ef2HnWSVG/upoSrEb3isEiwUDAwTizPEs1jR4OSQk5J/ZBFmykXassR8giF4zOLO2OQvl0H9L8eMB35/nWqS+vUHb1AdGE45ElOQ974tFra8sydSAx32vF6IzEDEJ45Iex+w/q1sq5WqiEeGjQlcwQDxuFcY1N2OiMDwLG4pD+efUeolU3OdK1qmZ0jU8rg8Jt2d4CEw+4cu1NbbKHbcsTAGVbAqXKjpdPePbsAs2H4pFjF7XoKlSn0QCoHuv4YTAL+81UNfq3Se2HwJtiufHNq80J3mTnZdybqxKN/Xl/oOJX2QzwqAQyay4Roab88PUKh2BKAOVtqfo0njdfzxMwUoF0DAP1E+FXu6h2Fvirlqf43VbJpidit7vOfBgKh0qjlWiYC21NPGJOxV4zwElvE6m2UyxRJegrOd45eoYNaccSasqtvxGb+p9QhhWtEk3T8P1bg8rMukYDegFoSlVCRGF5v9+sT0+jVk+yrd71EN2axX+JeLVMfFWVLN49yf0LtFISECCe/JJGgnFfLMoRLKImju6XXDHm/uuEs7i+INfb8=
   - secure: jChdlwrKObp67AjFLxv4pybk4vZQ3DzkydcNflqzlaxs9FxoBi5AQ+d7d7cNAITFWpTppdt07UC7V6h7rFTVsu9G5gkpDfI5j2e3KtDo3yiI2+TwxZilh2ZEl15WqwarYMXxvlMdGyaa+emciqyhR/VG3PfBUy2ZuSa5/nI8gZjo8LDmfH0/a4QopiM5c1qoniFjxDI0XwXXXiYs5mHGbjTQxfe0raEYjVT5qqGzCBgPSIFRFjhRJcJ8ww7Xdsz9xlMwONWqQT5gl6Be+h4n/0NhNLFuO9lWkxpEquXIxBQjpnf7eISt7O5ENH2gg5fdl8fdQxt2YSPuPrLcNLrMHIphaC41kZFx+WxBmTP3Sbwi6Pk/uSYE2TBmPrLPDkHR8a8SdQ3w0KJA1nB2x5wxtonuDlORgWeHQRn5uecvqPkyUI+L2P/pmSC976lg9g8tGU6uok6XkFSGo42I56G/wjYdzbmCZOYdG2jXDaEocNMyRtiztrgoZZ1nNfI87vQJ2LLjZ86T+LwvQoU3RJN0TYCYF2MSP1++IChrhIdjVTQAb9AE+nesnUaVVI4wcdleSK1AhVyFUr0d5jbTgTwd1M1DwoScGDtF4Faozy0e7K7dOPYj2mAQiEqhoFF07lkQWW52F76QzgHYBVUeOGTxYRCulSJIoxBiEU7AXGvOY0Y=
+  - DOCKER_USERNAME=olutiab
+  - HEROKU_LOGIN=yomi.olutimehin@yahoo.com
+  - DOCKER_PASSWORD: 
+  - secure: oneR8r799EUXy9md8DT+9tYYm4Y87JFj2UbumWERBTyxNiIeI5vbk5NopHsLCbAWNBhaJkBfg7y98rCB4fST7aax4M1VmjiLMmfsWO1dXM245B4jK82v+ZAHM+OvIxr09vt6S/X+uvV21BheK0QQ/U1cFxYJRH+cu8XCDrGbnFI81wmvyv+UFULNWHk11d+SfvnsbYhVmXY+3EpyWa1dBeAad/F8g9QZ+/LtTWLczRFJvuRKIwBuQvEWrmXWa2y0DK0M+hb929tyhBY2YNTGtnuL85UQpTfXVmH4iqYRNhO6D6DUEhVyGl1mF8NA6jJRTynRdD267jAFz/WO8TlH9Kz5XxIOVDZW0OdzoiRKybHwS5Xpxf385sJc+lqQlu23ZFBAY1IeXg745U3Bj3u1QKTlGYTpmvg39bQQE0R3+LXtjDvUTDg3aGmCkSovCMO1ffQQ/fv+UycApsqpe108fHuZcspxpRoxW+OS23WJ1iu6IpKR96Kp5zem58JoKadOXpm9mfMCGevP8r6EvPjlRTQfaW5MoBk0Hw5/YHnH983FyZqkn6zo3wla0e5LWHLIsMq0jYzqXeLqFU0hjCB+OxJ9XrruJA+3yVncHDBugV8UGY7VsN1Cp5UDn/q+YLxgAClIk/Qmegyg1if2/2UB6W7/DDSArXKnb22xBXRuA/k=
```

Comment:
This is slightly different syntax to the HEROKU_API_KEY below, which means that travis is actually reading this twice - once as an empty string. That's not causing problems as the second valid & secure version is overriding the first, but would be worth fixing up for clarity